cdap-common/src/main/java/co/cask/cdap/common/UnauthenticatedException.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.common;
+
+/**
+ * Thrown when a user is not authenticated.
+ * Note: This extends {@link UnauthorizedException} for backwards compatibility.
+ */
+public class UnauthenticatedException extends UnauthorizedException {
```

Comment:
Is there a JIRA to change the clients?  Also, we might need Unauthorized exceptions right when we do authorization